src/components/mangaInfo/MangaInfoDetails.js

```diff
@@ -66,8 +79,8 @@ MangaInfoDetails.defaultProps = {
 };
 
 // Helper functions
-function detailsElements(mangaInfo: MangaType): React.Node {
-  const fieldNames = ['status', 'source', 'author', 'genres', 'categories'];
+function detailsElements(mangaInfo: Manga): React.Node {
+  const fieldNames = ["author", "artist", "genre"];
```

Comment:
I removed categories because in order to get that information with the new v3 API, we have to make another API call and since Tachiyomi doesn't display the categories on the manga info screen anyways, I didn't feel like that was needed in TachiWeb.

The categories information use to come with the manga-info API call but is no longer present in the API v3 version of that call. It was removed because it actually involves making another DB query (separate from the one that gets the manga info) which although fairly cheap in other applications, is expensive in TachiWeb due to Tachiyomi's architecture.

Basically, all the v3 endpoints are designed in such a way that `one API call = about one DB query`.

Reply:
That said, yeah, I agree it should be refactored, I'll move the detail section into it's own component.